public/lib/loadSwaggerUI.js

```diff
@@ -16,6 +16,7 @@ $(function() {
 
   var accessToken;
   function loadSwaggerUi(config) {
+    var methodOrder = ['get', 'head', 'put', 'post', 'delete'];
```

Comment:
May as well add it defensively, I don't think it should be before `get`. How about `['get', 'head', 'options', 'put', 'post', 'delete']`?